addons/project/static/src/components/project_right_side_panel/components/project_profitability.xml

```diff
@@ -1,86 +1,86 @@
 <?xml version="1.0" encoding="utf-8"?>
 <templates id="template" xml:space="preserve">
     <t t-name="project.ProjectProfitability" owl="1">
-        <div class="o_rightpanel_subsection" t-if="revenues.data.length">
-            <table class="w-100 table-striped">
+        <div class="o_rightpanel_subsection pb-3" t-if="revenues.data.length">
+            <table class="w-100 table table-striped">
                 <thead>
                     <tr>
-                        <th style="width: 40%">Revenues</th>
-                        <th style="width: 20%; text-align: right;">Invoiced</th>
-                        <th style="width: 20%; text-align: right;">To Invoice</th>
-                        <th style="width: 20%; text-align: right;">Expected</th>
+                        <th>Revenues</th>
+                        <th class="text-end">Invoiced</th>
+                        <th class="text-end">To Invoice</th>
+                        <th class="text-end">Expected</th>
                     </tr>
                 </thead>
                 <tbody>
                     <tr t-foreach="revenues.data" t-as="revenue" t-key="revenue.id" t-if="revenue.invoiced !== 0 || revenue.to_invoice !== 0">
                         <t t-set="revenue_label" t-value="props.labels[revenue.id] or revenue.id"/>
                         <td>
-                            <a t-if="revenue.action" class="o_rightpanel_button"
+                            <a t-if="revenue.action" href="#"
```

Comment:
Using an href attribute is not mandatory ([ref](https://www.w3.org/TR/2016/REC-html51-20161101/textlevel-semantics.html#the-a-element))
>  If the [a](https://www.w3.org/TR/2016/REC-html51-20161101/textlevel-semantics.html#elementdef-a) element has no [href](https://www.w3.org/TR/2016/REC-html51-20161101/links.html#element-attrdef-a-href) attribute, then the element [represents](https://www.w3.org/TR/2016/REC-html51-20161101/dom.html#represent) a placeholder for where a link might otherwise have been placed, if it had been relevant, consisting of just the element’s contents.

... unless you needed the href for style purposes?

Reply:
It was for styling purpose exactly 

In _reboot.scss : 

`a:not([href]):not([class]) {
  &,
  &:hover {
    color: inherit;
    text-decoration: none;
  }
}`